docs/guides/OPENAI_CHAT_COMPLETION.md

```diff
@@ -98,7 +98,7 @@ var result =
 String resultMessage = result.getContent();
 ```
 
-### Using a Custom Resource Group
+## Using a Custom Resource Group
```

Comment:
(Comment)

Why the drive-by changes 😅

Reply:
this ## adds underlines which makes it more readable because the file is super long, it also aligns with the other files.